sympy/printing/latex.py

```diff
@@ -1077,7 +1077,15 @@ def _print_ceiling(self, expr, exp=None):
 
     def _print_log(self, expr, exp=None):
         if not self._settings["ln_notation"]:
-            tex = r"\log{\left(%s \right)}" % self._print(expr.args[0])
+            if len(expr.args) == 2:
```

Comment:
These two if checks are the wrong way round.

Can you add a test for what happens when printing a 2-arg log if the `ln_notation` flag is set in the printer?

Reply:
@oscarbenjamin ,
I'm not sure I do understand you. Are you asking to develop a test case for an expression like log(x,y, ln_notation=True) ?